network-api/networkapi/buyersguide/templatetags/bg_selector_tags.py

```diff
@@ -37,13 +37,12 @@ def track_record(value):
     about the possible options, and the context in which to apply
     this tag, rather than a generic "localize" tag.
     """
-    lcontext = "This is a rating for a company's history concerning privacy"
     if value == 'Great':
-        return pgettext(lcontext, 'Great')
+        return pgettext("This is a rating for a company's history concerning privacy", 'Great')
```

Comment:
this makes me sad =(

Hurray that it fixes it, but the fact that we need this unwrapped is deeply disappointing

Reply:
Yeah, this is a little mind-blowing that this doesn’t work…